### ISSUE TITLE
Need to clarify more on character death

### ISSUE THREAD

**Issue by zmoazeni** [our-stories/west-marches#28]
Some conversation [on discord](https://discordapp.com/channels/740929841997545492/740929842421170207/741068755710509167).

> Yeah that's been a fuzzy topic that I haven't addressed. But I've thought about it several times.
>
> I made a point to ask that in the session issue. I think character death (and difficulty in general) should be handled in a session-by-session basis. Because differences in that really affect the encounters and challenges IMO.

**Comment by crimsonknave** [comment]
IMO character death is much less of an issue in settings where Raise Dead is accessible, even if that's the party working to get the funds for it. As levels go up it becomes more and more likely that a character can be brought back to life. And if you're fighting something that can cause insta death (disintegration ray or whatnot) that just raises the stakes and it's very unlikely they'll be one shot.

**Comment by djigneo** [comment]
The ideal outcome for me is that we find a way to make death feel meaningful, rather than just a speed bump on the path toward immortality.
I wouldn't hold it against a player for wanting a beloved character resurrected, but I don't really want it to be a flippant, everyday occurence either. At this point I'm mostly debating tone more than mechanics, really.

**Comment by crimsonknave** [comment]
That's reasonable. There certainly are limits in 5e that IMO make it impactful. 
* `Revifify` is the cheapest and lowest level, but only works within 1 minute, so you'd have to have it prepared and have 300GP of materials ready at hand.
* `Raise Dead` only works within 10 days and requires 500GP. Depending on where you die you might not be within 10 days travel of someone who can cast it or the materials. And we have to be 9th level or find a 9th level Cleric or Bard to pay to cast it.
* `Reincarnate` Similar to Raise Dead, but it also changes your race, so it's going to have lasting repercussions
Once you get to 13th level the limitations really fall away, but that's the area where you start encountering things that can disintegrate you and make you unable to be resurrected (Ask me how I know :stuck_out_tongue_winking_eye: )

**Comment by zmoazeni** [comment]
One simplistic thought I have is that if we follow DnD death save mechanics, and a character dies in an adventure,  perhaps their character "limps" back to town.

- No XP from the adventure (or perhaps only what they earned up to that challenge)
- No treasure or renown
- Maybe they take a -1 (or -2) to a random stat for the next adventure they're on.

The only thing that would stink is that we'll have a player in the session just watching the remainder of it. And that can be unfun.

I really would prefer to avoid the harshness of not understanding the rules, or action economy strategies of the game. The more serious character death is, the more that tends to be promoted. 

But even if its a matter of the _wounded_ character still survives but doesn't benefit from the adventure. That may be enough incentive to  avoid dying. Or promote running the fuck away from baddies. Or not derping head long into battle.

Thoughts?

**Comment by zmoazeni** [comment]
Also. I do think the session/party should be able to _request_ a .one intense session where they _can't_ limp away.

I would also expect more risk, more reward too. XP, treasure, AND renown.

**Comment by crimsonknave** [comment]
IMO the Death Save rules are pretty lenient. If we're not starting at level 1 then in my experience you only die if:
1. You were really stupid
2. Your party is so out matched/out of position that someone can't heal you in 3-5 turns
3. The GM is being mean and has you take damage every turn (environment or monsters) which makes you fail a save

If the other members of the party can't make it to you in 3-5 turns to `Spare the Dying` or some healing something is really wrong IMO

**Comment by zmoazeni** [comment]
@crimsonknave I do largely agree with your sentiments, however I want to add some context here on my reasoning.

**I want to _strongly_ encourage newcomers to this community.** One of the biggest challenges, aside from their personal time, is the fear of screwing things up. What could be worse than screwing things up for yourself, where you have to spend more time spinning up a new character AND possibly cause similar issues with the party you're with?

**I would like folks to be comfortable starting/joining sessions with people they have never met.** This can totally get weird if a stranger is jeopardizing _MY_ character. I'd like to avoid those types of incentives/pressures if I can.

**I want players to know ahead of time if they may risk their character to the point where they can't play them in a future session.** It's a bit of a gamble right? Each session my character has _a chance_ they could perish. I'd like to be up front about that and allow the GM to tailor the session appropriately.

![](https://media.tenor.com/images/cb091d42a85b92171531127739025729/tenor.gif)

**I want GMs to be comfortable to try new things AND to make mistakes.** I don't want to punish GMs either. _Especially NEW ones._

So that's why I lean towards the default session is "limp your way back home." With the option that during the session setup the party can choose "Fuck yeah, we got this! Bring it on!" and then as a GM I can 😈 without shooting to JUST aiming to wipe them out.

---

So in games you GM and play in, by all means! State that's your expectations! Hell I would love to both GM AND Play in those same games.

But I wouldn't want that to be a blanket expectation for all games that flow through. I would like to also GM AND Play with folks (newcomers or not) where the threat level is so low that we're not stressing about dying, we just want to have a blast and joke around.

**Comment by crimsonknave** [comment]
Those are all good points. I think it'll be good to write up a "Welcome to Our Stories" doc that spells some of that out. It should also have some helpful tips like "Don't forget running away is always an option (though, you may need to cover your retreat)" and "Some places are more dangerous than others, if there are rumors of a Dragon living in a cave, not everyone might survive if you go in there unprepared"

**Comment by zmoazeni** [comment]
Yeah 100%. Also should be on the GM to softly remind players in the course of the game too. A "How to run the game" resource also feels appropriate to coach GMs as well.

I love this discussion ❤️

**Comment by djigneo** [comment]
If we're talking about homebrew-style rules, one policy I've heard goes something like:
Death saves are secret to the player that makes them. Only when another creature examines the dying creature does it know if it's dead, doing saves, or stable. So, a player who is performing death saves can essentially "fudge" the rolls if they wish no questions asked (or they can play it RAW).

**Comment by crimsonknave** [comment]
At the very least having then secret should motivate players to go and save them!

**Comment by crimsonknave** [comment]
One other thing that has ocurred to me. I expect that almost all of our adventures will start with the party having had a Long Rest and will end with the party returning to town for another (unless we are continuing). This dramatically shifts the resources that are available to spend in taking monsters down and keeping party members alive. This will likely shift our encounters to be more of a few hard/deadly encounters instead of half a dozen moderate where we're rationing spells and abilities. 

I'm not sure how it'll play out, but I'd expect us to be more prepared for tough encounters and thus less likely to die than if we'd had a bunch of little encounters and were unprepared for a big tough one.

**Comment by zmoazeni** [comment]
@crimsonknave Yeah you and I are mostly aligned.

* **All adventures start fully fresh, unless a continuation**
* **You all can choose to take short rests as you like**
* **You all can also take long rests, but we'll roll for an encounter which may disrupt it.** - I'll have to look at some encounter tables for this. I would imagine that the longer you spend in an area, the harder it will be to get a fully uninterrupted long rest.

I may sprinkle in some small encounters and see about the balance. The one thing that is nice with fewer hard/deadly encounters is that the party can pick and choose how to tackle them. It also feels easier to balance.

But starting the initiative clock every time can become tedious for sure.

**Comment by crimsonknave** [comment]
It's actually pretty hard to have something interrupt a long rest according the the PHB `If the rest is interrupted by a period of strenuous activity — at least 1 hour of walking, fighting, casting spells, or similar adventuring activity — the characters must begin the rest again to gain any benefit from it.` As I read that you can fight for 59 minutes including casting spells and then go back to sleep and get your rest. So, unless there's an extended standoff or a long chase I think you can mostly just complete your rest.

**Comment by zmoazeni** [comment]
Yeah. I should have clarified.

**I need to figure out a solid long rest encounter % formula.** I'm currently thinking it starts at 20%. For the first long rest 20% a random  encounter occurs. Then the 2nd long rest would be be adding 1/4th of that percent and rounding up. So:

1. `20%`
2. `20/4+20=25%`
3. `25/4+25=32%`

I may need to tweak this. But the longer you stay in an area, the higher likelihood something will be tracking you.

With this idea, short rests don't change the % at all. And this is _per area._ The party may start in one area, and move to explore another. In that 2nd, the rate drops back to 20% again.

**Even if an encounter occurs, the party will benefit from the long rest after the fight.** so its not a punishment. It's a potential price to pay.

Now the thing that could change this is if lets say one of the members is dragged away and kidnapped (in the course of combat. Grappling or knocked unconscious). The party would need to decide to pursue, and cancel the long rest and its benefits.

**Comment by crimsonknave** [comment]
So, I think a few guard rails will solve this for the moment.

* GMs should advertise ahead of time that death is possible/this is potentially really dangerous if they think it will be
* GMs should pause the game and make a note that a character might die if they think an action might be more dangerous than the players thought (for example if players are just not picking up hints that there's an angry dragon in the cave).
* If the party is unable/unwilling to resurrect any deceased character the player is free to have their character be brought back by an NPC or organization who they now owe a great debt to.

If that seems good to folks I think we can add those notes into the repo and close this.

**Comment by djigneo** [comment]
I concur with points 1 & 2. I agree with the spirit of 3.

I guess I would suggest treating this like we are treating other things. We can say our recommendations / guidelines and let players do what they think is fun. Correct course later if we need to.
My thoughts are summed up above - I think death should be impactful. I think it's fine to come back to life if it makes the story more interesting.

**Comment by zmoazeni** [comment]
I like it being settled with the GM/players when the session is scheduled. How about we add that to the session checklist?
